Code/EnginePlugins/GameComponentsPlugin/Gameplay/Implementation/RaycastComponent.cpp

```diff
@@ -54,10 +55,16 @@ EZ_BEGIN_COMPONENT_TYPE(ezRaycastComponent, 3, ezComponentMode::Static)
     EZ_MEMBER_PROPERTY("ForceTargetParentless", m_bForceTargetParentless),
     EZ_BITFLAGS_MEMBER_PROPERTY("ShapeTypesToHit", ezPhysicsShapeType, m_ShapeTypesToHit)->AddAttributes(new ezDefaultValueAttribute(ezVariant(ezPhysicsShapeType::Default & ~(ezPhysicsShapeType::Trigger)))),
     EZ_MEMBER_PROPERTY("CollisionLayerEndPoint", m_uiCollisionLayerEndPoint)->AddAttributes(new ezDynamicEnumAttribute("PhysicsCollisionLayer")),
-    EZ_MEMBER_PROPERTY("CollisionLayerTrigger", m_uiCollisionLayerTrigger)->AddAttributes(new ezDynamicEnumAttribute("PhysicsCollisionLayer")),
-    EZ_ACCESSOR_PROPERTY("TriggerMessage", GetTriggerMessage, SetTriggerMessage),
+    EZ_MEMBER_PROPERTY("ChangeNotificationMsg", m_sChangeNotificationMsg),
   }
   EZ_END_PROPERTIES;
+  EZ_BEGIN_FUNCTIONS
+  {
+    EZ_SCRIPT_FUNCTION_PROPERTY(GetCurrentDistance)->AddFlags(ezPropertyFlags::Const),
```

Comment:
There is no need to explicitely specify the const flag if the function itself is const already. Then the const flag is already added by our reflection automatically.